src/searchbar.cpp

```diff
@@ -208,16 +208,16 @@ void SearchBarLineEdit::onScroll(int value)
         return;
     }
 
-    /* m_completer.popup()->currentIndex() being invalid means the user used the
+    /* m_suggestionView->currentIndex() being invalid means the user used the
        keyboard up/down to scroll past the end.
     */
-    if (!m_completer.popup()->currentIndex().isValid())
-        return onScrollPastEnd();
+    if (!m_suggestionView->currentIndex().isValid())
+        onScrollPastEnd();
```

Comment:
The `return` statement was lost (though if key down is handled in `SearchBarLineEdit::eventFilter()` this piece of code will simply have to be deleted).

Reply:
Done